linters/go.mod

```diff
@@ -0,0 +1,6 @@
+module github.com/otterize/intents-operator/linters
+
+// versions here has to be the same as the versions used by golangci-lint
+go 1.20
```

Comment:
But... this says 1.21 😱 
https://github.com/otterize/intents-operator/pull/457/files#diff-107e910e9f2ebfb9a741fa10b2aa7100cc1fc4f5f3aca2dfe78b905cbd73c0d2R42